src/server/tiering/disk_storage.cc

```diff
@@ -106,6 +111,10 @@ std::error_code DiskStorage::Stash(io::Bytes bytes, StashCb cb) {
   if (offset < 0) {
     size_t start = io_mgr_.Span();
     size_t grow_size = -offset;
```

Comment:
Something I noticed here, it's a `size_t` and you assigned `-offset`. Is this intented?

Reply:
yes, because the if (offset < 0) one line above 🙂

Reply:
I am blind :heart_eyes: :rofl: